src/core/textrenderer/qgstextbackgroundsettings.cpp

```diff
@@ -275,6 +302,10 @@ QColor QgsTextBackgroundSettings::fillColor() const
 void QgsTextBackgroundSettings::setFillColor( const QColor &color )
```

Comment:
Shouldn't the getters also default to trying to retrieve the property from the fill symbol?

Reply:
@nyalldawson , I don't think so, because it can eventually get out of sync (the fill color is still used by the SVG background so if someone would be to switch back and forth between rectangle and SVG, he/she could have two different colors)

Reply:
Ok, fair enough